features/integration-github-bitbucket-gitlab.md

```diff
@@ -14,3 +14,60 @@ You can then either create a new organization directly synchronized with a GitHu
 Apart from synchronizing teams, users, and permissions, setting up the integration will simplify the addition of new packages to your Composer repository. When you create a **new repository** on GitHub, Bitbucket, or GitLab, it will be **added as a Composer package automatically** if it contains a _composer.json_ file. If you’d like to add existing repositories as packages, you can do so with the click of a button on the Packages tab in your organization.
 
 **Update web hooks** notifying Packagist of new code pushes will be installed in the background, and **credentials to access the package** will be **configured automatically**.
+
+## Integrations
+
+Private Packagist integrates with the following systems:
+
+#### GitHub
+* OAuth: Users authenticate on Private Packagist with their GitHub accounts
+* Synchronization:
+    * Keeps team members and access permissions in sync with your GitHub organisations
+    * Synced repositories use the default repository permission on GitHub
+* Credentials: GitHub API Token
+* Webhooks: Code changes, releases, created repositories, team creation or member changes
+
+#### GitHub Enterprise
+* OAuth: Users authenticate on Private Packagist with their Bitbucket accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/github-integration-setup.md) for enterprise.
+* Synchronization:
+    * Keeps team members and access permissions in sync with your GitHub Enterprise organisations
+* Credentials: GitHub API Token
+* Webhooks: Code changes, releases, created repositories, team creation or member changes
+
+#### Bitbucket.org / Cloud
+* OAuth: Users authenticate on Private Packagist with their Bitbucket.org accounts
+* Synchronization:
+    * Keeps users permissions in sync with your Bitbucket groups
+    * Group and user changes aren't kept in sync
+* Credentials: Bitbucket API Key or Bitbucket App Password
+* Webhooks: Code changes
+
+#### Bitbucket / Stash Server
+* OAuth: Users authenticate on Private Packagist with their Bitbucket accounts. Please contact us to set this up for the cloud plan or follow [this setup](../docs/enterprise/bitbucket-server-integration-setup.md) for enterprise.
+* Synchronization:
+    * Keeps users and access permissions in sync with your Bitbucket projects
+    * Individual collaborators aren't supported
+    * New projects or team changes aren't kept in sync
+* Credentials: SSH keys
+* Webhooks: Code changes
+
+#### GitLab
+* OAuth: Users authenticate on Private Packagist with their GitLab accounts
+* Synchronization:
+    * Keeps access permissions in sync with your Gitlab groups
+    * Individual collaborators aren't kept in sync
+    * New projects or team changes aren't kept in sync
+* Credentials: GitLab API token
+* Webhooks: Code changes
```

Comment:
We also get notified about releases. Actually pretty sure we get notified for changes + releases on every platform